backend/src/seeders/seeders.service.ts

```diff
@@ -24,7 +24,7 @@ export class SeederService {
 		private readonly userRelService: UserRelationshipService,
 	) {}
 	// on my local machine the src doesn't exist in this path
-	private readonly shouldSeedFilePath = './dist/src/seeders/.hasSeeded';
+	private readonly shouldSeedFilePath = './.hasSeeded';
 
```

Comment:
is that problematic?

Reply:
This has been solved, the comment is outdated and now removed.